### PR TITLE
doc: Added notification into this guide to help developer solve "dock…

### PR DESCRIPTION
**Pull request by xingularity**
Added notifications into CONTRIBUTORS_USER_QUICKSTART to help developers when they encounter docker credential error.

### CODE REVIEW

File: CONTRIBUTORS_QUICK_START.rst [hunk 1]
```diff
@@ -287,6 +287,15 @@ Setting up Breeze
 
   $ ./breeze --python 3.8 --backend mysql
 
+Note: If you encounter an error like "docker.credentials.errors.InitializationError:
+docker-credential-secretservice not installed or not available in PATH", you may execute following command to fix it.
```
Comment (uranusjr) [inline CONTRIBUTORS_QUICK_START.rst]:
```suggestion
docker-credential-secretservice not installed or not available in PATH", you may execute the following command to fix it:
```

File: CONTRIBUTORS_QUICK_START.rst [hunk 2]
```diff
@@ -1101,6 +1110,15 @@ Setting up Breeze
...
```
Comment (uranusjr) [inline CONTRIBUTORS_QUICK_START.rst]:
```suggestion
.. note::

    If you encounter an error like "docker.credentials.errors.InitializationError:
    docker-credential-secretservice not installed or not available in PATH", you may
    execute the following command to fix it.
```

File: CONTRIBUTORS_QUICK_START.rst [hunk 3]
```diff
@@ -1101,6 +1110,15 @@ Setting up Breeze
...
+
+.. code-block:: bash
+
+  $ sudo apt install golang-docker-credential-helper
```
Comment (uranusjr) [inline CONTRIBUTORS_QUICK_START.rst]:
```suggestion
    $ sudo apt install golang-docker-credential-helper
```

I think 2 spaces is not enough? (Not sure)

**Review by uranusjr — COMMENTED** [review]
~~I am wondering if we can rewrite the first two notes into one so the same content is not repeated twice.~~

Edit: The repeated content is under PyCharm and VSCode setup instructions so I guess it's fine.

**Comment by uranusjr** [comment]
Also: Could you reformat the commit message to conform to the general Git commit format?

See: https://robertcooper.me/post/git-commit-messages

File: CONTRIBUTORS_QUICK_START.rst [hunk 4]
```diff
@@ -1110,14 +1112,15 @@ Setting up Breeze
 
   $ ./breeze --python 3.8 --backend mysql
 
-Note: If you encounter an error like "docker.credentials.errors.InitializationError:
-docker-credential-secretservice not installed or not available in PATH", you may execute following command to fix it.
+.. note::
+   If you encounter an error like "docker.credentials.errors.InitializationError:
+   docker-credential-secretservice not installed or not available in PATH", you may execute the following command to fix it:
```
Comment (uranusjr) [inline CONTRIBUTORS_QUICK_START.rst:1117]:
```suggestion
.. note::

   If you encounter an error like "docker.credentials.errors.InitializationError:
   docker-credential-secretservice not installed or not available in PATH", you may execute the following command to fix it:
```

I _think_ an empty line is needed? (Feel free to ignore this if Sphinx renders it correctly for you.)

Comment (potiuk) [reply CONTRIBUTORS_QUICK_START.rst:1117]: Actually for Contributors quick start we should rely on what GitHub does for rendering. We do not build it as part of our documentation

Comment (uranusjr) [reply CONTRIBUTORS_QUICK_START.rst:1117]: Never mind this then. GitHub renders fine.

**Review by uranusjr — APPROVED** [review]

**Pull request merged**